src/interfaces/appstore.js

```diff
@@ -172,6 +172,12 @@ module.exports = function(app) {
 
     getModulesInfo(plugins, getPlugin, all)
     getModulesInfo(webapps, getWebApp, all)
+
+    if ( process.env.DISABLED_PLUGIN_UPDATES ) {
+      let disabled = process.env.DISABLED_PLUGIN_UPDATES.split(' ')
```

Comment:
Why space and comma for example? Just seems unnatural to me.

const ?

Reply:
I'm ok with either, space just seems more "shell" like. For example, a `for` loop in a shell script uses space as the separator.

Reply:
I'll leave this to you.